src/i18n/en.json

```diff
@@ -0,0 +1,138 @@
+    "log_in": "Log In",
+    "no_more_resend": "You haven't received the Email yet? We have already forwarded it to you, if it doesn't arrive in the next few minutes, please contact support",
+    "placeholder_email": "Your Email will be your Username",
+    "placeholder_firstname": "Name",
+    "placeholder_lastname": "Lastname",
+    "placeholder_password": "Enter your secret key",
+    "placeholder_phone": "9 11 2345-6789",
+    "privacy_policy_consent_HTML": "I accept Doppler's <a target=\"_blank\" href=\"https://www.fromdoppler.com/en/privacy-policy?utm_source=app&utm_medium=landing&utm_campaign=signup\">Privacy Policy</a>.",
+    "promotions_consent": "Sign me up for promotions about Doppler and its partners.",
+    "resend_email": "Resent it",
+    "sign_up": "Sign Up",
+    "sign_up_sub": "No contracts or credit cards required, up to 500 Subscribers.",
+    "thanks_for_registering": "Thank you for registering"
+  },
+  "upgradePlanForm": {
+    "message_placeholder": "Your message",
+    "plan_select": "Select Plan",
+    "title": "Request an update of your plan"
+  },
+  "validation_messages": {
+    "error_account_is_blocked_invalid_pass_HTML" : "For security reasons we've temporarily disabled your account. <a href=\"http://www.fromdoppler.com/en/contact\" target=\"_blank\">Contact Us.<a>",
+    "error_account_is_canceled_HTML" : "Your account has been cancelled. To know more please <a target=\"_blank\" href=\"http://www.fromdoppler.com/en/contact.html\">contact us</a>.",
+    "error_checkbox_policy": "Ouch! You haven't accepted the Doppler's Privacy Policy.",
+    "error_email_already_exists": "Ouch! You already have a Doppler account.",
+    "error_invalid_domain_to_register_account": "Ouch! Invalid Email to create an account.",
+    "error_invalid_email_address": "Ouch! Invalid email address",
+    "error_invalid_login" : "Ouch! There is an error in your Username or Password. Please, try again.",
+    "error_password_character_length": "At least 8 characters",
+    "error_password_digit": "One number",
+    "error_password_letter": "One letter",
+    "error_password_safe": "Your password is safe!",
+    "error_phone_invalid": "Ouch! The phone number is not valid",
+    "error_phone_invalid_country": "Ouch! The country code is not valid",
+    "error_phone_too_long": "Ouch! The phone number is too long",
+    "error_phone_too_short": "Ouch! The phone number is too short",
+    "error_required_field": "Ouch! The field is empty.",
+    "error_unexpected": "Unexpected error. Please try again or contact support."
```

Comment:
`Support`?

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
@cbernat is right.